public/app/features/canvas/elements/button.tsx

```diff
@@ -19,6 +20,12 @@ interface ButtonConfig {
   api?: APIEditorConfig;
 }
 
+const defaultApiConfig: APIEditorConfig = {
+  endpoint: '',
+  method: HttpRequestMethod.GET,
```

Comment:
Considering the original used method was post perhaps is better to leave it as default?

Reply:
@academo Defaulting to GET felt more natural as it's the first option and I didn't think would cause any confusion as the method was not exposed in the UI. But probably it's just safer, I'll change it. 

As for the previously configured panels - the defaults should be applied only when we're adding a new button (`getNewOptions`) so nothing else should be impacted.